src/main/kotlin/it/unibo/lss/fcla/consulting/domain/consulting/Date.kt

```diff
@@ -0,0 +1,38 @@
+package it.unibo.lss.fcla.consulting.domain.consulting
+
+/**
+ * @author Stefano Braggion
+ *
+ * Value object representing a date in the consulting context
+ */
+data class Date(val year: Int, val month: Int, val day: Int) {
+
+    companion object {
+        const val MIN_YEAR: Int = 1900
+        const val MIN_MONTH: Int = 1
+        const val MAX_MONTH: Int = 12
+        const val MIN_DAY: Int = 1
+        const val MAX_DAY: Int = 31
```

Comment:
February has 28 days. You should have used a library to handle Date. Maybe fix this in a future release